docs/components/control/authenticate-with-callback.mdx

```diff
@@ -141,7 +141,7 @@ The `<AuthenticateWithRedirectCallback />` is used to complete a custom OAuth fl
 
 | Name | Type | Description |
 | --- | --- | --- |
-| `afterSignInUrl` | `string` | Full URL or path to navigate to after successful sign in. Defaults to `/`. It's recommended to use [the environment variable](/docs/deployments/clerk-environment-variables#sign-in-and-sign-up-redirects) instead. |
+| `afterSignInUrl` (legacy) | `string` | Full URL or path to navigate to after successful sign in. Defaults to `/`. It's recommended to use [the environment variable](/docs/deployments/clerk-environment-variables#sign-in-and-sign-up-redirects) instead. **`signInFallbackRedirectUrl` has priority over the legacy `afterSignInUrl`. Use `fallbackRedirectUrl` or `forceRedirectUrl` instead of  `afterSignInUrl`.** |
```

Comment:
How about using `deprecated` instead of `legacy`?

Reply:
yes, I agree, it should say deprecated